tests/playwright/sanity/core/app/modules/onboarding/onboarding.test.ts

```diff
@@ -56,7 +56,7 @@ test.describe( 'On boarding @onBoarding', async () => {
 
 		expect( popup.url() ).toContain( 'my.elementor.com/signup' );
```

Comment:
*Not for this PR*
Maybe we can mock this page, it's part of elementor.com, not part of the Elementor plug-in